py/selenium/webdriver/remote/webdriver.py

```diff
@@ -741,6 +744,59 @@ def execute_async_script(self, script, *args):
             'script': script,
             'args': converted_args})['value']
 
+    def load_brython(self,
+                     src="https://cdn.jsdelivr.net/npm/brython@3.8.9/brython.min.js",
```

Comment:
I also thought of something similar to what you suggested but unfortunately it can be unreliable.  For example, in Chrome whenever I try to access local files I get the follow error `Not allowed to load local resource: {file name here}`. However, I believe in Firefox it would work with no problem (I have not tested this but I have reason to believe this to be the case from the exhaustive amount of research I have conducted on this topic).  Additionally, unless configured in a certain way, Brython has a reliance on certain external files anyway.  Loading from a file would not change the need for the internet to access the external files.

Reply:
We need to use bazel here. I will help with this part.